docs/deciding_what_data_to_stream_with_dfe_analytics.md

```diff
@@ -0,0 +1,70 @@
+# Deciding what data to stream with ```dfe-analytics```
+To ensure compliance with relevant legislation it is **imperative** that ```dfe-analytics``` users:
+- Only stream personal data to BigQuery for which the Department has a legal basis for processing under GDPR
+- Only retain this data for as long as this legal basis remains valid
+
+It is also **recommended** that teams use 'hidden field' functionality to hide any more sensitive fields that are streamed, such as protected category data and human-readable Personally Identifiable Information (PII).
+
+This guide sets out how teams should decide what data to stream with ```dfe-analytics``` and links to resources to explain how to do this.
+
+```dfe-analytics``` is designed to be used alongside the [```dfe-analytics-dataform``` Dataform package](https://github.com/DFE-Digital/dfe-analytics-dataform/) to transform streamed data. The approach below relies on using both together.
+
+## What is personal data?
+Almost **all** data that is useful for data analysis in Teacher Services is personal data. Without being able to identify people it is not possible to count the things that are most important for analysis in Teacher Services, because these 'things' are people: teachers, teacher training candidates, trainee teachers, prospective teachers we are in contact with etc.
+
+There are some common myths about what is and is not personal data. These are not true:
+- "Only human readable personal data is personal data - email addresses, names etc." - in fact, anything that uniquely identifies an individual is personal data. Database identifiers, IP addresses etc. can all be personal data even if it is not possible for a human to identify who some data belongs to.
+- "Replacing identifiers of individuals with other identifiers stops it being personal data" - this is a process known as pseudonymisation. This process may improve data security in some situations, but does not stop data being personal data. See [ICO guidance](https://ico.org.uk/for-organisations/uk-gdpr-guidance-and-resources/personal-information-what-is-it/what-is-personal-data/what-is-personal-data/#pd4) for more information.
```

Comment:
The ICO guidance https://ico.org.uk/for-organisations/uk-gdpr-guidance-and-resources/personal-information-what-is-it/what-is-personal-data/what-is-personal-data/#pd4 includes an example. Suggest we provide examples relevant to teacher services for the the key parts of the guidance

Reply:
I've added examples to the second two myths; the first already had them.